examples/inverse-calculations.py

```diff
@@ -0,0 +1,63 @@
+#%%
+import resdx
+import numpy as np
+from scipy import optimize
+from scipy.optimize import fsolve
+import random
+import matplotlib.pyplot as plt
+import seaborn as sns
+sns.set()
+
+def plot(x, y, xlabel, ylabel):
+    plt.figure()
+    plt.plot(x, y)
+    plt.xlabel(xlabel)
+    plt.ylabel(ylabel)
+    plt.savefig(f'output/{ylabel.replace(" ", "")}_vs_{xlabel.replace(" ", "")}.png')
+
+# Cooling 
+
+def fan_efficacy(seer):
+    if seer <= 14:
+        return resdx.u(0.25,'W/(cu_ft/min)')
+    elif seer >= 16:
+        return resdx.u(0.18,'W/(cu_ft/min)')
+    else:
+        return resdx.u(0.25,'W/(cu_ft/min)') + (resdx.u(0.18,'W/(cu_ft/min)') - resdx. u(0.25,'W/(cu_ft/min)'))/2.0 * (seer - 14.0) 
+
+
+def c_d(seer):
+    if seer <= 12:
+        return 0.2
+    elif seer >= 13:
+        return 0.1
+    else:
+        return 0.2 + (0.1 - 0.2)*(seer - 12.0)
+
+# Heating
+
+def estimated_seer(hspf): # Linear model fitted (R² = 0.994) based on data of the histrory of federal minimums (https://www.eia.gov/todayinenergy/detail.php?id=40232#).
+    return (hspf - 3.2627)/0.3526
+
+#%% Cooling inverse calculations with constraints
+seer_range = np.arange(6,26.5,0.1)
+cops_from_seer = []
+for seer in seer_range:
+    root_fn = lambda x : resdx.DXUnit(gross_cooling_cop_rated = [x], fan_eff_cooling_rated = [fan_efficacy(seer)], c_d_cooling=c_d(seer)).seer() - seer
+    cop, solution = optimize.newton(root_fn,seer/3.33, full_output = True)
+    #print(f"seer = {seer:.1f}, cop = {cop:.2f}, fan_eff = {resdx.convert(fan_efficacy(seer),'W/(m**3/s)','W/(cu_ft/min)'):.3f}, C_d = {c_d(seer):.3f}, converged = {solution.converged}, iter = {solution.iterations}")
+    cops_from_seer.append(cop)
+
+plot(seer_range, cops_from_seer, "SEER", "Gross (fan not included) COP (at A conditions)")
```

Comment:
I still think it's redundant to include "(fan not included)". Just leave it as "Gross". Same on heating plot.

Reply:
I misinterpreted your previous comment. I thought you meant it's redundant but it's ok because it will help others who may find gross vs net is confusing to better understand the difference.